maths/average_mode.py

```diff
@@ -0,0 +1,18 @@
+def mode(inputlist):                    					#Defining function "mode."
+	"""
+	This function returns the mode(Mode as in the measures of central tendency) of the input data.
+	
+	>>> mode([2,3,4,5,3,4,2,5,2,2,4,2,2,2])
+	2
+	
+	the input list may contain any Datastructure or any Datatype.

```

Comment:
Put this line above the doctest (and Capitalize the first letter).

Let's add another test...
```
>>> input_list = [2,3,4,5,3,4,2,5,2,2,4,2,2,2]
>>> mode(input_list)
2
>>> import statistics
>>> mode(input_list) == statistics.mode(input_list)
True
```